### PR TITLE
1st commit

### PR DESCRIPTION
**Pull request by JersonEM**
add breadcrumb's components:
- [slug].astro
- about.astro
- mockData.js
- Breadcrumbs.test.js
- Breadcrumbs.stories.js
- Breadcrumbs.astro
- ArticlePage.astro
- ArticleInfo.test.js
- ArticleInfo.stories.js
- ArticleInfo.astro
- Article.astro

### CODE REVIEW

**Comment by DanX3N0** [comment]
repair the pipe is failing

File: src/components/ArticleInfo.astro [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  .article-info__item {
+    margin-bottom: 0.5rem;
+  }
+  .article-info__label {
+    font-weight: bold;
+  }
+  .article-info__value {
+    margin-left: 0.5rem;
+  }
+</style>
```
Comment (Ditmar) [inline src/components/ArticleInfo.astro:34]: the component should be made in react

**Review by Ditmar — CHANGES_REQUESTED** [review]
The components must be made following the structure that was proposed, you can't come and get such nonsense out of nowhere, repeat everything you did, everything is wrong in this pr.

**Review by Ditmar — CHANGES_REQUESTED** [review]
The pipe is broken this won't be a review

**Review by Ditmar — CHANGES_REQUESTED** [review]
image
No estas siguiendo las directivas y practicas establecidas

File: src/pages/index.astro [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+
+const article = {
+  date: '2024-06-13',
+  volume: '42'
+};
 ---
 
+<ArticleInfo date={article.date} volume={article.volume} />
+
 <Layout title="Welcome to Astro.">
```
Comment (Ditmar) [inline src/pages/index.astro:13]: remove this code

File: package-lock.json
Comment (Ditmar) [inline package-lock.json:1]: dont' change this code

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Comment by Ditmar** [comment]
I don't see activity in this pr I will close

**Pull request closed without merging**